src/main/java/page_objects/ChangePasswordPage.java

```diff
@@ -0,0 +1,15 @@
+package page_objects;
+
+import helper.element_helper.Label;
+import org.openqa.selenium.By;
+
+public class ChangePasswordPage extends GeneralPage {
+
+    //Elements
+    private final Label lblChangePassword = new Label(By.cssSelector("[align='center']"));
```

Comment:
for title in this page, we should use //div[@id='content']//h1.
If there are more than one locator that have attribute `[align='center']`, your locator will not be unique

Reply:
Thanks for your review. I just have fixed and pushed into this branch. Please review my newest update.